dojo/models.py

```diff
@@ -422,6 +422,12 @@ class System_Settings(models.Model):
         verbose_name=_('Enable Remediation Advice'),
         help_text=_("Enables global remediation advice and matching on CWE and Title. The text will be replaced for mitigation, impact and references on a finding. Useful for providing consistent impact and remediation advice regardless of the scanner."))
 
+    enable_similar_findings = models.BooleanField(
+        default=True,
```

Comment:
Keep in mind that this feature is enabled, and as I understand it can affect performance

Reply:
yes , the idea is parameterise it system settings.